pymc/distributions/continuous.py

```diff
@@ -828,6 +850,12 @@ def dist(cls, sigma=None, tau=None, sd=None, *args, **kwargs):
 
         return super().dist([0.0, sigma], **kwargs)
 
+    def get_moment(rv, size, loc, sigma):
+        moment = loc + sigma
```

Comment:
I have a doubt here.

Current implementation: loc + sigma
[Wikipedia version](https://en.wikipedia.org/wiki/Half-normal_distribution): loc + \sqrt{\frac{2}{\pi}}sigma

Reply:
I think you are right, we should probably be using that. Do you want to open a PR to fix it?

Reply:
Sure, happily :)

Reply:
A bit more involved, we should probably also use the truncated normal mean as the moment. We might even be able to simplify the switch statement, if the normal log_cdf behaves well with +- infinity values: https://en.wikipedia.org/wiki/Truncated_normal_distribution